error when running derive changeset after changing poi search radii

```
Feb 16 18:23:56 ip-10-0-1-201.ec2.internal server[17962]: Caused by: java.lang.RuntimeException: Error parsing params json string.  mapId = 1041
Feb 16 18:23:56 ip-10-0-1-201.ec2.internal server[17962]: at hoot.services.utils.DbUtils.getConflationType(DbUtils.java:495) ~[classes/:?]
Feb 16 18:23:56 ip-10-0-1-201.ec2.internal server[17962]: at hoot.services.controllers.grail.GrailResource.createChangeset(GrailResource.java:310) ~[classes/:?]

```

--- a/hoot-services/src/main/java/hoot/services/utils/DbUtils.java
+++ b/hoot-services/src/main/java/hoot/services/utils/DbUtils.java
@@ -488,7 +488,7 @@ public class DbUtils {
             JSONObject expectedObj;
 
             try {
-                sourceInfo = sourceInfo.replace("\\", ""); // have to unescape
+                sourceInfo = sourceInfo.replace("\\", "").replace("\"{", "{").replace("}\"", "}"); // have to unescape
                 expectedObj = (JSONObject) parser.parse(sourceInfo);
             }
             catch (ParseException e) {

